src/main/webapp/techList.html

```diff
@@ -35,10 +35,14 @@
                         <a href={{redirectUrl(tech.id)}}>
                             <h3>{{tech.name}}</h3>
                         </a>
-                        <p class='technologydescription'>{{tech.shortDescription}}</p>
+                        <p>{{tech.shortDescription}}</p>
                     </div>
                 </div>
             </div>
         </div>
     </div>
+    <div style="width: 50%; text-align: center; margin: 0 auto;">
```

Comment:
Please don't use inline CSS. It's a bad practise. Import an external stylesheet instead.